son-gtklic/licenseManager/settings.py

```diff
@@ -0,0 +1,10 @@
+
+
+SQLALCHEMY_DATABASE_URI = 'postgresql://es:es-test@192.168.0.201:5432/usermanagement'
```

Comment:
Please do not assume this. Environment variables but be passed to each docker, check son-gtksrv (e.g.), it also uses PostgreSQL as DB